library-model-loader/src/main/resources/edu/ucr/cs/riple/librarymodel/nullable-methods.tsv

```diff
@@ -1,2 +0,0 @@
-test.target.Foo	returnNullableBad(int)
-test.target.Foo	returnNullableGood(int)
```

Comment:
Why are re removing this library model TSV? (Just want to understand, since it seems unrelated to the renaming and it isn't called out in the PR description)

Reply:
`LibraryModelsLoader` module here is used for unit tests and these are left over contents from previous runs. They shouldn't appear in the git repository.